### PR TITLE
Add sig-compute owner string to relevant tests

### PR DESCRIPTION
**Pull request by vatsalparekh**
**What this PR does / why we need it**:
Add sig-compute owner string to relevant tests so that a separate test lane can filter and run these specific tests.
The tests that seemed to relate the core virt components, like VM, VMI, VNC and more are marked here with `sig-compute` owner tag

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

**Special notes for your reviewer**:

**Release note**:

```release-note
NONE
```

### CODE REVIEW

File: tests/infra_test.go
Comment (davidvossel) [inline tests/infra_test.go:69]: owner set twice

Comment (vatsalparekh) [reply tests/infra_test.go]: Corrected

**Review by davidvossel — COMMENTED** [review]
/approve
/hold

The PR looks fine except for a random ` tests/k8s-reporter/1_events.log` file. Maybe it slipped in? 

I can add a lgtm and remove the hold once that file is removed.

**Comment by vatsalparekh** [comment]
/cc @rmohr @fgimenez 
> /approve
> /hold
> 
> The PR looks fine except for a random ` tests/k8s-reporter/1_events.log` file. Maybe it slipped in?
> 
> I can add a lgtm and remove the hold once that file is removed.

There are some failures after the retest as well, but related to the flakes, changes here are unrelated

**Review by fgimenez — COMMENTED** [review]
Changes look good, but we need also changes in the logic to filter out the marked tests, something like https://github.com/kubevirt/kubevirt/blob/master/automation/test.sh#L48-L50 (without the CNAO env var) and https://github.com/kubevirt/kubevirt/blob/master/automation/test.sh#L298-L299

**Comment by fabiand** [comment]
@iholder-redhat please review this PR

@vatsalparekh can you please add a sentence to the PR description how you made the decision to mark these testcases to be owned by compute?

**Comment by vatsalparekh** [comment]
> Changes look good, but we need also changes in the logic to filter out the marked tests, something like https://github.com/kubevirt/kubevirt/blob/master/automation/test.sh#L48-L50 (without the CNAO env var) and https://github.com/kubevirt/kubevirt/blob/master/automation/test.sh#L298-L299

@fgimenez How if we just add `KUBEVIRT_E2E_FOCUS` variable similar to this https://github.com/kubevirt/project-infra/pull/1059/files#diff-26f7cd0389085c1da47f919a02c4dbeca5ddd96ca00d8e49c0862f698475aaf9R176

**Comment by vatsalparekh** [comment]
@fgimenez @davidvossel this seems ready for review

**Comment by vatsalparekh** [comment]
Again, the flakes here seem unrelated

**Comment by fgimenez** [comment]
> @fgimenez How if we just add `KUBEVIRT_E2E_FOCUS` variable similar to this https://github.com/kubevirt/project-infra/pull/1059/files#diff-26f7cd0389085c1da47f919a02c4dbeca5ddd96ca00d8e49c0862f698475aaf9R176

@vatsalparekh Sure, sounds good to me

**Review by fgimenez — APPROVED** [review]
/lgtm

**Review by davidvossel — APPROVED** [review]
/hold cancel

**Pull request merged**
